polling_stations/apps/data_collection/management/commands/import_derby.py

```diff
@@ -48,24 +27,11 @@ def address_record_to_dict(self, record):
         ):
             rec["postcode"] = "DE24 0LR"
 
-        if (
-            record.addressline1.strip() == "Flat 2"
-            and record.addressline2.strip() == "70 Wilkins Drive"
-            and record.addressline3.strip() == "Allenton"
-            and record.addressline4.strip() == "Derby"
-        ):
-            rec["postcode"] = "DE24 8LU"
-
-        if uprn == "10010687582":
-            rec["postcode"] = "DE22 4LT"
```

Comment:
This one you've removed here is the correction for your `Discarding record: Postcode centroid is outside target local authority`

Reply:
Now reinstated. Thanks for the spot!